src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -253,157 +223,94 @@ const formConfig = {
       title: 'Medicare information',
       pages: {
         hasMedicareAB: {
-          path: ':index/medicare-ab',
-          arrayPath: 'applicants',
-          showPagePerItem: true,
-          title: item => `${applicantWording(item)} Medicare status`,
-          CustomPage: ApplicantMedicareStatusPage,
-          CustomPageReview: ApplicantMedicareStatusReviewPage,
+          path: 'medicare-ab',
+          title: formData => `${nameWording(formData)} Medicare status`,
```

Comment:
This change is applied to just about all pages in the Medicare section (removing the loop specific properties).